--- cmake/modules/FindUMFPACK.cmake.old	2015-01-11 10:45:10.824107696 +0100
+++ cmake/modules/FindUMFPACK.cmake	2015-01-11 10:45:42.600776211 +0100
@@ -11,7 +11,9 @@
 set(CMAKE_LIBRARY_PATH ${BLAS_DIR}/lib $ENV{BLAS_DIR}/lib ${CMAKE_LIBRARY_PATH})
 find_package(AMD QUIET)
 find_package(BLAS QUIET)
+if (DOLFIN_ENABLE_CHOLMOD)
 find_package(CHOLMOD QUIET)
+endif()
 
 # Check for header file
 find_path(UMFPACK_INCLUDE_DIRS umfpack.h
